app/models/mixins/retirement_mixin.rb

```diff
@@ -39,6 +39,10 @@ def set_retirement_requester(obj_ids, requester)
     end
   end
 
+  def instance_retire_request
+    make_retire_request(id, User.current_user)
```

Comment:
1.  You need to call class method so - `self.class.instance_retire_request` otherwise it fails
2. Can you keep signature from class method ?
3. Can you use same name as class method have ?

Reply:
Shouldn't need the splat since using it this way will only ever be on a single vm, I'm pretty sure.

Reply:
@d-m-u Have you tried to run this method on single  vm in console ?
`Vm.first.instance_retire_request`

and what about point 2 and 3 ?

Reply:
I already addressed them?